server/index.js

```diff
@@ -276,6 +276,40 @@ app.get("/api/all-todos", async (req, res) => {
   }
 });
 
+/**
+ * REST API #8 for changing todo status
+ * @description
+ * - Extracts id from request
+ * - Extracts isDone from request
+ * - Updates todo status in database
+ * - Gets the updated todo information
+ * - If success,
+ *  - Sends todo information
+ * - If fails,
+ *  - Sends 400 status
+ * @param {object} req HTTP request object
+ * @param {object} res HTTP response object
+ */
+app.post("/api/check-todo", async (req, res) => {
+  try {
+    const id = req.body.id;
+    const isDone = req.body.isDone;
+
```

Comment:
Only allow the **creator** of this todo to change its status.

If the user who requested this API is not the creator of this todo, let the request fail.